recipes/utide/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.1.0" %}
+{% set commit = "0c612d557f615fbbb7d513892b6840b2302b65e7" %}
+
+package:
+    name: utide
+    version: {{ version }}.dev0
+
+source:
+    fn: {{ commit }}.tar.gz
+    url: https://github.com/wesleybowman/UTide/archive/{{ commit }}.tar.gz
+    sha256: 1000aa9c552309e8e4f5a20f9ed2e62cd1c6bc3794476f5aacc25c2326d6fe08
+
+build:
+    number: 0
+    skip: True  # [win and np>=111]
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - numpy
+        - scipy 0.16.0  # [win]
```

Comment:
We need to pin `scipy to 0.16.0` on Windows due to a bug when reading MatlabTM (`.mat`) files. There are some reports that this is a bug in the anaconda scipy since other scipy sources do work just fine.  Ping @msarahan.